### PR TITLE
WIP, started updating rules to fit new version

### PR DESCRIPTION
**Pull request by bleehu**

### CODE REVIEW

File: Rules Combat.html [hunk 1]
```diff
@@ -34,7 +35,7 @@ <h2>Moving</h2>
                 <p class="rules"> 
                     Each Mech has a movement drive, which gives a speed measured in 
                     spaces. On each of your turns, if you chose to move, you may move
-                    up to that number of spaces on unobstructed terrrain. For example,
```
Comment (bleehu) [inline Rules Combat.html:37]: facepalm. I can't believe I wrote this.

**Review by bleehu — COMMENTED** [review]
I approve all of 1Source's changes :)

File: Rules Combat.html (hunk 1 above)
Comment (1sourcecontrol) [reply Rules Combat.html:37]: typ[s happen to the best of us ;)

**Review by 1sourcecontrol — APPROVED** [review]
Let er' rip!

**Pull request merged**